src/components/ClockModule.js

```diff
@@ -147,137 +150,145 @@ function ClockModule() {
       />
 
       <HeaderPopover
-        animateHeight={false}
         heading="Court clock"
-        height={(isSynced ? 27.5 : 26) * GU}
-        width={45 * GU}
+        direction={DEFAULT_DIRECTION}
         onClose={handlePopoverClose}
         opener={buttonRef.current}
+        screenData={{
+          courtConfig,
+        }}
+        screenId={DEFAULT_SCREEN_ID}
+        screenKey={({ courtConfig }) => courtConfig?.id}
```

Comment:
We might be able to find a better key for this, but I'm also wondering if this really needs to use `HeaderPopover` since it doesn't transition between screens.

Maybe we should keep these two popovers (`ClockModulePopover` and `AccountModulePopover`) separate?